src/main/webapp/WEB-INF/view/searchResults.jsp

```diff
@@ -38,10 +39,11 @@
 	<ol class="list-unstyled col-md-8">
 		<c:forEach var="doc" varStatus="s" items="${documentItems}">
 			<li id="result${s.index}">
-				<h3 class="title text-truncate">
+				<h3 class="title text-truncate" style="margin-bottom:0;">
```

Comment:
It's better not to use a style attribute.

Reply:
It was an unnecessary STYLE, so we removed it.
Sorry for the unnecessary code.